appengine/accessible/levelmanager.component.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * Blockly Demos: AccessibleBlockly
```

Comment:
Blockly Games: Accessible
And all other similar files (not counting those being removed since they are copies of Accessible Blockly).